db/seeds/questions.yml

```diff
@@ -130,7 +162,41 @@ be_humble:
   question_type: long_text
   word_limit: 100
 
-# Additional Info
+# Background
+
+fa20_technical_gauge:
+  title: '[Optional] If you are applying as a developer, from the following courses, list out the ones you have taken/are currently taking:'
+  hint: "- CS 10\n- CS 61A\n- Data 8\n- CS 88\n- CS 61B\n- Atleast 1 CS upper div course (answer yes or no)\n- Other: (could include relevant decals such as the Web Design/React/Rails Decals or other learning experiences)"
+  required: false
+  question_type: short_text
+  question_category: background
+
+fa20_classes_additional_info:
+  title: '[Optional] Feel free to add anything else we should know about your answer to the above (ex. currently auditing, applied for WDD, took a class but dropped partway due to COVID).'
+  hint: There is no need to elaborate on the classes you have taken beyond the ones on the list above (for example, we don't need to specific upper div courses). Please limit your response to 100 words.
+  required: false
+  question_category: background
+  question_type: long_text
+  word_limit: 100
+
+fa20_background_question:
+  title: '[Optional] Is there any additional information or circumstances we should know about to understand you better? This can be anything related to your identity, current/previous access to relevant resources, a personal situation due to COVID, etc. In reading your application, we want to ensure that we understand and respect your background, experiences, and perspectives.'
```

Comment:
Change to "Are there any additional circumstances you’d like to us to know about to better understand you?" 
just for grammar